js/views/message_view.js

```diff
@@ -57,13 +57,19 @@
         render_attributes: function() {
             var seconds = this.model.get('expirationTimerUpdate').expireTimer;
             var name = this.conversation.getTitle();
+            var timerMessage;
             if (this.conversation.id === textsecure.storage.user.getNumber()) {
-                name = i18n('you');
+                timerMessage = {
+                  content: i18n('youChangedTheTimer',
+                             Whisper.ExpirationTimerOptions.getName(seconds))
+                };
+            } else {
+                timerMessage = {
+                  content: i18n('theyChangedTheTimer', [name,
+                             Whisper.ExpirationTimerOptions.getName(seconds)])
+                };
             }
-            return {
-              content: i18n('changedTheTimer', [name,
-                         Whisper.ExpirationTimerOptions.getName(seconds)])
-            };
+            return timerMessage;
```

Comment:
`grep`d around and followed this style https://github.com/WhisperSystems/Signal-Desktop/blob/master/js/views/error_view.js#L27-L33 instead of two returns.